src/lints/pub_api_sealed_trait_became_unsealed.ron

```diff
@@ -0,0 +1,54 @@
+SemverQuery(
+    id: "pub_api_sealed_trait_became_unsealed",
+    human_readable_name: "Sealed trait in the public API became unsealed",
+    description: "An public API sealed trait is now unsealed, allowing downstream crates to implement it. Reverting this in the future would be a breaking change",
```

Comment:
```suggestion
    description: "A public API sealed trait has become unsealed, allowing downstream crates to implement it. Reverting this would be a major breaking change.",
```